pubspec.yaml

```diff
@@ -29,3 +30,4 @@ dev_dependencies:
   node_preamble: "^1.0.0"
   test: "^0.12.0"
   yaml: "^2.0.0"
+  bootstrap_sass: ^4.0.0-alpah.3+4
```

Comment:
I don't like making assertions about other packages' assets in tests. I'd rather create a folder in the test body, then create a `.packages` file that points to it.